src/python/pants/backend/python/goals/lockfile.py

```diff
@@ -147,6 +147,10 @@ async def generate_lockfile(
                     *(f"--find-links={link}" for link in req.find_links),
                     *pip_args_setup.args,
                     *req.interpreter_constraints.generate_pex_arg_list(),
+                    *(
```

Comment:
Is this the kinda thing worth a test? Or is it somehow used in our bootstrapping?